pkg/sql/sqlbase/system.go

```diff
@@ -24,23 +24,31 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 )
 
-// ShouldSplitAtID determines whether a specific descriptor ID
-// should be considered for a split at all. If it is a database
-// or a view table descriptor, it should not be considered.
-func ShouldSplitAtID(id uint32, rawDesc *roachpb.Value) bool {
+// ShouldSplitAtDesc determines whether a specific descriptor should be
+// considered for a split. If it is a database or a view table descriptor,
```

Comment:
This comment could just say that only plain tables are considered for splits

Reply:
Done.